osbs/api.py

```diff
@@ -200,10 +200,18 @@ def get_build_logs(self, build_id, follow=False, namespace=DEFAULT_NAMESPACE):
             if ex.status_code != 404:
                 raise
         else:
-            if build["status"] in ["Complete", "Failed"]:
-                return build["metadata"]["labels"]["logs"]
-            else:
-                return self.os.logs(build_id, follow=False, namespace=namespace)
+            build_response = BuildResponse(build)
+            # Should this be 'build_response.is_finished()'?
```

Comment:
`["Complete", "Failed"]` doesn't seem to match any of the `osbs.constants.BUILD_*_STATES` values. Are there states missing from that `get_build_logs()` check, or should there be a new state list constant to cover `["Complete", "Failed"]`?